setup/src/Magento/Setup/Module/Di/Code/Reader/FileScanner.php

```diff
@@ -38,6 +38,14 @@ protected function scan()
             define('T_TRAIT', 42001);
         }
 
+        // ensure php backwards compatibility (from laminas code 3.5.x)
+        if (! defined('T_NAME_QUALIFIED')) {
+            define('T_NAME_QUALIFIED', 24001);
+        }
+        if (! defined('T_NAME_FULLY_QUALIFIED')) {
+            define('T_NAME_FULLY_QUALIFIED', 24002);
+        }
```

Comment:
Let's format all if statements in single format - space isn't needed here.
```suggestion
        if (!defined('T_NAME_QUALIFIED')) {
            define('T_NAME_QUALIFIED', 24001);
        }
        if (!defined('T_NAME_FULLY_QUALIFIED')) {
            define('T_NAME_FULLY_QUALIFIED', 24002);
        }
```

Reply:
@ihor-sviziev, it’s not ready for review yet. I try to mark it as “dev in progress” every time.

Reply:
Ah, that's label for issues. The correct label is "needs update". Also you can convert PR to draft

Reply:
Thank you, will do the Draft next time.